src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -298,6 +298,8 @@ namespace winrt::TerminalApp::implementation
 
             _settings = std::move(newSettings);
 
+            _settings.ExpandCommands();

```

Comment:
I believe it'd be better to lazily expand them. I think we're currently not lazy loading the command palette so this would not be effective right now anyways, but I'd prefer if we'd work towards making the command palette load lazily. Loading it is the most CPU intensive thing we do during startup (apart from loading WinUI in general).

Especially given that `ExpandedCommands()` is only called once, and thus doesn't need to cache anything, I believe that this can be removed without replacement.